cfgov/scripts/tests/test_smoke_tests.py

```diff
@@ -36,55 +36,87 @@ def test_home_page_test_failure(self, mock_get):
 class HttpTests(unittest.TestCase):
     """Tests for the http smoke tests"""
 
+    @mock.patch('scripts.http_smoke_test.requests.get')
+    def test_get_full_list(self, mock_get):
+        mock_response = mock.Mock()
+        mock_response.json.return_value = {
+            'top20': [']url1'],
+            'apps': ['url2']
+        }
+        mock_get.return_value = mock_response
+        full_list = http_smoke_test.get_full_list()
+        self.assertEqual(len(full_list), 2)
+        self.assertEqual(mock_get.call_count, 1)
+
+    @mock.patch('scripts.http_smoke_test.requests.get')
+    def test_get_full_list_fallback(self, mock_get):
+        """Check that script falls back to hard-coded list."""
+        mock_get.side_effect = ValueError
+        full_list = http_smoke_test.get_full_list()
+        self.assertEqual(len(full_list), len(http_smoke_test.FULL_RUN))
+
     @mock.patch('scripts.http_smoke_test.requests.get')
     def test_http_success_short(self, mock_get):
         mock_response = mock.Mock()
         mock_response.status_code = 200
         mock_get.return_value = mock_response
         http_smoke_test.check_urls('pro1')
-        self.assertEqual(mock_get.call_count, len(http_smoke_test.SHORT_RUN))
+        self.assertEqual(
+            mock_get.call_count,
+            len(http_smoke_test.SHORT_RUN)
+        )
 
     @mock.patch('scripts.http_smoke_test.requests.get')
-    def test_http_success_full(self, mock_get):
+    @mock.patch('scripts.http_smoke_test.get_full_list')
+    def test_http_success_full(self, mock_list, mock_get):
+        mock_list.return_value = http_smoke_test.FULL_RUN
         mock_response = mock.Mock()
         mock_response.status_code = 200
         mock_get.return_value = mock_response
         http_smoke_test.check_urls('pro1', full=True)
```

Comment:
Let's check the return value of the smoke tests, at least on the tests that check the "success" path.

```suggestion
         result = http_smoke_test.check_urls('pro1', full=True)
         self.assertTrue(result)
```

Reply:
Yes!